src/Game/Systems/PlayerDeathSystem.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Systems/PlayerDeathSystem.h"
+
+PlayerDeathSystem::PlayerDeathSystem(World* m_World, EventBroker* eventBroker)
+    : System(m_World, eventBroker)
+{
+    EVENT_SUBSCRIBE_MEMBER(m_OnPlayerDeath, &PlayerDeathSystem::OnPlayerDeath);
+}
+
+void PlayerDeathSystem::Update(double dt)
+{
+}
+
+bool PlayerDeathSystem::OnPlayerDeath(Events::PlayerDeath& e)
+{
```

Comment:
If you have time, create an XML entity to represent the death animation with everything that's needed (camera, model, animation, explosion effect, lifetime) instead of having the creation hardcoded here.